src/js/components/DataFilter/__tests__/DataFilter-test.tsx

```diff
@@ -271,6 +273,30 @@ describe('DataFilter', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('range inclusive', () => {
+    const { getByRole, container } = render(
+      <Grommet>
+        <Data data={[{ age: 1 }, { age: 2 }, { age: 3 }, { age: 4 }]}>
+          <DataFilters>
+            <DataFilter range={{ min: 1, max: 4, step: 1 }} property="age" />
+          </DataFilters>
+          <DataTable />
+        </Data>
+      </Grommet>,
+    );
+
+    const lowerBound = screen.getByRole('slider', { name: 'Lower Bounds' });
+    act(() => {
+      lowerBound.focus();
+    });
+    fireEvent.keyDown(lowerBound, { key: 'Right', keyCode: 39 });
+    const applyFiltersButton = getByRole('button', { name: 'Apply filters' });
+    fireEvent.click(applyFiltersButton);
+
+    expect(screen.queryByText('1')).not.toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.queryByText('1')).not.toBeInTheDocument();
    expect(screen.queryByText('2')).toBeInTheDocument();
```
add this to ensure the "inclusive" aspect is checked?

Reply:
Added this in latest commit, I also removed the snapshot since I think the other assertions are effectively testing the behavior we want